src/cascadia/WpfTerminalControl/WpfTerminalControl.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+  <PropertyGroup>
+    
+    <TargetFrameworks>net472</TargetFrameworks>

```

Comment:
Is 4.7.2 the right framework version to go back as far as you wanted here? I thought 4.7.2 was relatively recent.

Reply:
4.7.2 is relatively recent but has been pushed out to all versions of windows back to windows 7. VS 2019 actually requires/installs 4.7.2.

Reply:
OK that's fine. Just wanted to make sure this was considered.